LICENCE.md

```diff
@@ -4,8 +4,19 @@ The MIT License (MIT)
 Copyright (c) 2016 Node.js Foundation
```

Comment:
This could be bumped to 2017 now...

ref: https://github.com/nodejs/TSC/issues/195